sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2179,9 +2182,14 @@ def provision_with_retries(
 
 
 class CloudVmRayResourceHandle(backends.backend.ResourceHandle):
-    """A pickle-able tuple of:
+    """A pickle-able handle for the cluster created by CloudVmRayBackend.
+
+    The handle object will last for the whole lifecycle of the cluster.
 
     - (required) Cluster name.
+    - (required) Cluster name on cloud (different from the cluster name, as we
+        append user hash to avoid confliction across multiple accounts in a same
+        organization, and truncate the name for length limit).
```

Comment:
```suggestion
        append user hash to avoid conflict b/t multiple users in the same
        organization/account, and truncate the name for length limit).
```